### PR TITLE
Ubiquiti Edgepower battery values

### PR DESCRIPTION
**Pull request by jepke**
Please give a short description what your pull request is for

New SNMP values:
ubntPsuBatteryChargeLevel
ubntPsaBatteryTimeRemaining
ubntPsuBatteryReplaceIndicator

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/discovery/edgeos-ep.yaml [hunk 1]
```diff
@@ -33,6 +33,21 @@ modules:
...
+            data:
+                -
+                    oid: ubntPsuBatteryChargeLevel
+                    num_oid: '.1.3.6.1.4.1.41112.1.5.3.2.1.9.{{ $index }}'
+                    index: 'ubntPsuBatteryChargeLevel.{{ $index }}'
+                    descr: 'Charge percentage PSU {{ $index }}'
+        runtime:
+            data:
+                -
+                    oid: ubntPsaBatteryTimeRemaining
```
Comment (Jellyfrog) [inline includes/definitions/discovery/edgeos-ep.yaml:46]: Misspelled

**Comment by murrant** [comment]
@jepke you need to fix the typo in the yaml, then you likely need to add new test data too.

**Comment by jepke** [comment]
Just saw Ubiquiti has a typo in their MIB 'ubntPsuBatteryTimeRemining' is that the typo you mean ?

**Comment by Jellyfrog** [comment]
I have commented on the exact line thats misspelled, `ubntPsaBatteryTimeRemaining`

**Comment by jepke** [comment]
@Jellyfrog thanks guess i need some glasses as i totally didnt see Ps**a**

**Comment by Jellyfrog** [comment]
Still need to re-generate testdata.

**Comment by jepke** [comment]
I ran the tests but it didn't change anything ill try it again.

**Comment by murrant** [comment]
@jepke you will need to run the collect step again because the data isn't included in the snmprec because the oid is wrong :D

**Review by murrant — APPROVED** [review]

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-69-release-changelog-november-2020/14124/1

**Pull request merged**